test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs

```diff
@@ -74,6 +74,61 @@ await foreach (Page<BinaryData> page in allPages.AsPages())
             }
         });
 
+        [Test]
+        public Task HandwrittenModelLro() => Test(async (host) =>
+        {
+            using var diagnosticListener = new ClientDiagnosticListener("dpg_customization_LowLevel", asyncLocal: true);
+            CollectionAssert.IsEmpty(diagnosticListener.Scopes);
+
+            Operation<Product> lro = await new DPGClient(Key, host).LroValueAsync(WaitUntil.Started, "model");
+            Assert.AreEqual(1, diagnosticListener.Scopes.Count);
+            Assert.AreEqual("DPGClient.LroValue", diagnosticListener.Scopes[0].Name);
+            Assert.IsTrue(diagnosticListener.Scopes[0].IsCompleted);
+
+            await lro.WaitForCompletionAsync();
+            Assert.AreEqual(2, diagnosticListener.Scopes.Count);
```

Comment:
Can we validate that the two scopes are siblings and not nested?

Reply:
No, as I said, the only indirect way to check that is that previous scope is closed before `WaitForCompletionAsync` is closed.

Reply:
@JoshLove-msft @christothes, FYI -- this is one of the gaps in our tests we were discussing yesterday.  Looks like we aren't able to validate a hierarchical relationship using our test infra.